packages/localize/src/date/parseDate.js

```diff
@@ -2,9 +2,9 @@ import { localize } from '../localize.js';
 import { getDateFormatBasedOnLocale } from './getDateFormatBasedOnLocale.js';
 import { addLeadingZero } from './addLeadingZero.js';
 
-const memoize = (fn, parm) => {
+const memoize = fn => {
   const cache = {};
-  return () => {
+  return parm => {
```

Comment:
we could also make this function more generic by doing `const n = [...arguments]` in the function above

Reply:
Do you think passing multiple parameters to function will improve it?

Reply:
Currently it's only used for this purpose. If it needs to be a generic memoize function (which the name suggests, and it makes sense to promote it to at some later point) then yes.
For now it's fine ;)